pages/linux/debsecan.md

```diff
@@ -0,0 +1,24 @@
+# debsecan
+
+> A tool to list vulnerabilities on a particular Debian installation.
+> More information: <https://gitlab.com/fweimer/debsecan>.
+
+- List vulnerable installed packages on the current host:
+
+`debsecan`
+
+- List vulnerable installed packages of a specific suite:
+
+`debsecan --suite {{release_code_name}}`
+
+- List only fixed vulnerabilities of testing ("sid"):
+
+`debsecan --suite {{sid}} --only-fixed`
```

Comment:
I find it confusing (`sid` seems to be [the code name of "unstable Debian"](https://wiki.debian.org/DebianUnstable)). What about:
```suggestion
- List only fixed vulnerabilities:

`debsecan --suite {{release_code_name}} --only-fixed`
```

Reply:
I'd include `- List only fixed vulnerabilities of a specific release`